.github/workflows/stale.yml

```
@@ -0,0 +1,23 @@
+name: "Stale label"
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  branches:
+    - dev-es    
+jobs:
+  stale:
+    timeout-minutes: 1
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v7
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been labeled with inactivity.'
+        stale-pr-message: 'This PR is stale because it has been labeled with inactivity.'
```

```suggestion
        stale-issue-message: 'This issue/PR is stale because it has been inactive for 60 days. This issue/PR will be closed automatically with additional 20 days of inactivity. If an update/comment occur on stale issue/PR, the stale label will be removed and the timer will restart. Closed issue/PR can be reopened if necessary.'
        stale-pr-message: 'This issue/PR is stale because it has been inactive for 60 days. This issue/PR will be closed automatically with additional 20 days of inactivity. If an update/comment occur on stale issue/PR, the stale label will be removed and the timer will restart. Closed issue/PR can be reopened if necessary.'
```

I think we'd better to provide full context on the stale and automatic close policy to contributors.